### PR TITLE
Pmm 4778 update mongodb exporter

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -25,7 +25,7 @@
 [submodule "mongodb_exporter"]
 	path = sources/mongodb_exporter/src/github.com/percona/mongodb_exporter
 	url = https://github.com/percona/mongodb_exporter.git
-	branch = 0.10.0
+	tag = 0.11.0
```
Comment (AlekSi) [inline .gitmodules:28]: `.gitmodules` do not support tags, that's why we used a branch.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-803-378ff9d
client docker - perconalab/pmm-client-fb:PR-803-378ff9d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-803-378ff9d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-803-30de582
client docker - perconalab/pmm-client-fb:PR-803-30de582
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-803-30de582.tar.gz

**Comment by BupycHuk** [comment]
So it's actually was tested as part of PMM-5661, https://github.com/Percona-Lab/pmm-submodules/pull/784

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-803-30de582
client docker - perconalab/pmm-client-fb:PR-803-30de582
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-803-30de582.tar.gz

**Pull request merged**